### PR TITLE
Add article attachment link

### PR DESCRIPTION
**Pull request by actlikewill**
image

### CODE REVIEW

File: peachjam/views/article.py [hunk 1]
```diff
@@ -79,6 +80,23 @@ def get_context_data(self, **kwargs):
...
+            ArticleAttachment,
+            document__date=self.kwargs["date"],
+            document__slug=self.kwargs["slug"],
+            document__author__username=self.kwargs["author"],
+            pk=self.kwargs["pk"],
+            filename=self.kwargs["filename"],
+        )
+
+    def render_to_response(self, context, **response_kwargs):
+        return FileResponse(self.object.file, filename=self.object.filename)
```
Comment (longhotsummer) [inline peachjam/views/article.py:97]: can you add a long-lived caching header hear? eg 1 year, like our other static files?

Comment (actlikewill) [reply peachjam/views/article.py]: added

File: peachjam/admin.py [hunk 2]
```diff
@@ -1153,6 +1153,39 @@ class ArticleAttachmentInline(BaseAttachmentFileInline):
     model = ArticleAttachment
     extra = 1
 
+    def attachment_url(self, obj):
+        return reverse(
```
Comment (longhotsummer) [inline peachjam/admin.py:1157]: It strikes me that this is the `get_absolute_url` for this attachment object.

**Comment by longhotsummer** [comment]
Looks good. When we render the attachments in a list at the end of the article, do we need to use the new url?

File: peachjam/admin.py (hunk 2 above)
Comment (actlikewill) [reply peachjam/admin.py]: right. I've added it as such and changed the links including in the template

**Review by longhotsummer — APPROVED** [review]

**Pull request merged**